docs/source/backends/adios2.rst

```diff
@@ -160,6 +160,23 @@ Ignore the 30GB initialization phases.
 .. image:: ./memory_groupbased_nosteps.png
   :alt: Memory usage of group-based iteration without using steps
 
+
+Known Issues
+------------
+
+.. warning::
+
+   Nov 1st, 2021 (`ADIOS2 2887 <https://github.com/ornladios/ADIOS2/issues/2887>`__):
+   libfabric 1.6+ introduced a couple of breaking changes that break ADIOS2 SST (staging/streaming) workflows.
```

Comment:
It weren't really breaking changes on libfabric's side. ADIOS2 is just using some heuristics for fabric selection which happened to work in 1.6, but are working no longer. (Example: The fabric selection in ADIOS2 depends on the order in which libfabric reports available fabrics.)

Reply:
Good point, do you have a suggestion how to reformulate this? :)

Reply:
A more neutral wording might be something like:
```
The fabric selection in ADIOS2 has been written for libfabric 1.6. With newer versions of libfabric, the following workaround is needed to guide the selection of a functional fabric for RDMA support:
```

Reply:
Perfect, thanks a lot.